packages/shared/src/components/post/PostContent.tsx

```diff
@@ -148,14 +159,21 @@ export function PostContent({
               'mt-4 !typo-callout',
               isVideoType ? 'mb-4' : 'mb-8',
             )}
+            domain={
+              !isVideoType &&
+              showOriginDomain &&
+              post.domain.length && (
+                <TruncateText>
+                  From{' '}
+                  <ArticleLink title={post.domain} className="hover:underline">
```

Comment:
Title should be the post permalink

Reply:
No i wanted it to show domain for mobile truncated version.
Hope that makes sense? Let me know what you think?

(By default its the domain but its our redirector and on mobile you might not see the full domain)

Reply:
It's fine as is, as we don't expose the canonical URL.